docs/deployment/ecs.md

```diff
@@ -1,5 +1,7 @@
 # AWS ECS (Fargate) + ALB (Application Load Balancer)
 
+**⚠** When using Fargate or vanilla ECS, you should set the number of worker carefully (https://github.com/developmentseed/titiler/issues/119 and https://github.com/tiangolo/fastapi/issues/253).
```

Comment:
This would be perfect for an "admonition"

```
!!! warning
	When using Fargate or vanilla ECS...
```

![image](https://user-images.githubusercontent.com/15164633/94590157-7e4d6d80-0243-11eb-8558-13e859938d8b.png)

I'd also include something like "you should set # of workers carefully. Setting too high a number of workers could lead to extra charges due to a bug in fastapi"